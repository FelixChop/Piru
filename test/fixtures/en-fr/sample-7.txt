He whispered a perfunctory apology, hoping it would placate the crowd, but the tension remained palpable.
