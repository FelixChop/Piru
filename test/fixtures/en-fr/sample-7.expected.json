[
  {
    "word": "perfunctory",
    "lemma": "perfunctory",
    "definition": "superficiel, fait sans soin",
    "citation": "He whispered a perfunctory apology, hoping it would placate the crowd, but the tension remained palpable.",
    "to_learn": true
  },
  {
    "word": "placate",
    "lemma": "to placate",
    "definition": "apaiser",
    "citation": "He whispered a perfunctory apology, hoping it would placate the crowd, but the tension remained palpable.",
    "to_learn": true
  },
  {
    "word": "palpable",
    "lemma": "palpable",
    "definition": "perceptible, Ã©vident",
    "citation": "He whispered a perfunctory apology, hoping it would placate the crowd, but the tension remained palpable.",
    "to_learn": true
  }
]
